src/main/java/uk/ac/ebi/biostudies/controller/Authentication.java

```diff
@@ -82,14 +92,22 @@ private void sendRedirect(HttpServletResponse response, String returnURL, boolea
         }
     }
 
-    @RequestMapping(value = "/logout")
+    @RequestMapping(value = "/signout", method = RequestMethod.POST)
```

Comment:
let's keep the same endpoint i.e. `/logout` 
Some users are already using it programatically